src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -27,6 +27,7 @@
 import com.cloudbees.jenkins.GitHubWebHook;
 import com.cloudbees.plugins.credentials.CredentialsNameProvider;
 import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Just for "visibility": that method exists so that it can be used in the tests to mock.